projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid.integration.spec.ts

```diff
@@ -426,8 +426,8 @@ describe('IgxHierarchicalGrid Integration', () => {
             expect(summaryRow.children[0].offsetWidth).toEqual(expander.nativeElement.offsetWidth);
             expect(summaryRow.children[1].tagName.toLowerCase()).toEqual('igx-display-container');
 
-            // there should be indentation of the summaries
-            expect(summaryRow.children[0].className.indexOf(INDENT_LEVEL_CLASS) !== -1).toBe(true);
+             // there should be indentation of the summaries
+             expect(summaryRow.children[0].className.indexOf(INDENT_LEVEL_CLASS) !== -1).toBe(false);
```

Comment:
This class INDENT_LEVEL_CLASS - igx-grid__row-indentation--level-1, does not seem to be used anymore. It appears that the summaries indent is handled by another class - igx-grid__summaries-patch and its width is bound to summariesMargin. So checks that depend on old class should be changed.

Reply:
The check is intentional in this is case so it should not display indentation level element. 
It is still present in the summary row component.